SalesforceSDK/packages/CommonServiceLocator.1.3/lib/portable-net4+sl5+netcore45+wpa81+wp8/Microsoft.Practices.ServiceLocation.XML

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>Microsoft.Practices.ServiceLocation</name>
+    </assembly>
+    <members>
+        <member name="T:Microsoft.Practices.ServiceLocation.ActivationException">
```

Comment:
All the files in this folder are external libraries that we reference in the SDK. We need to check these in order to be able to reference these in a Cordova project as nuget does not work for a Cordova project yet.